docs/guide/units-coordinates.rst

```diff
@@ -229,7 +229,7 @@ Using Coordinates with SunPy Map
    >>> ax = plt.subplot(projection=m)  # doctest: +REMOTE_DATA
    >>> m.plot()  # doctest: +REMOTE_DATA
    <matplotlib.image.AxesImage object at ...>
-   >>> ax.plot_coord(c, 'o');  # doctest: +REMOTE_DATA
+   >>> _ = ax.plot_coord(c, 'o')  # doctest: +REMOTE_DATA
```

Comment:
I think this is potentially confusing for a user; would it be possible to add a descriptive variable name that captures what ax.plot_coord returns?

Reply:
The problem is that `plot_coord()` returns a value in Astropy 4.0, and doesn't return a value for earlier versions.  Add a descriptive variable name for a function that doesn't return anything would arguably be more confusing.

Reply:
It's not a widely used code pattern, admittedly, but people do use `_ = ...` to indicate when the return value is not actually going to be used, which is the case here.